src/components/navigation/header/drawer/userInfo/index.tsx

```diff
@@ -52,29 +57,26 @@ const DrawerUserInfo: FC<Props> = ({ user }) => {
           </Stack>
         </Stack>
         {selectedTenant ? (
-          <Stack direction="row">
-            <GarageIcon />
-            <Stack
-              direction={{
-                base: 'column',
-                md: 'row',
-              }}
-              spacing={{
-                base: 0,
-                md: 'sm',
-              }}
-            >
-              <Box as="span" fontWeight="bold">
-                {selectedTenant.billingName || selectedTenant.id}
-              </Box>
-              <Box as="span">
-                {selectedTenant.billingZipCode} {selectedTenant.billingCity}
-              </Box>
-            </Stack>
-          </Stack>
+          <>
+            <Show above="md">
+              <SelectedTenantInfo selectedTenant={selectedTenant} />
+            </Show>
+            <Show below="sm" marginX="-lg" marginBottom="-lg">
```

Comment:
not sure i understand it fully. what is shown between md and sm?

Reply:
Above MD (769px+) has the tenant selection menu in the header and therefore only tenant info is shown in the drawer
![Screenshot 2025-03-20 at 10 02 30](https://github.com/user-attachments/assets/8c23712f-fe9a-43d6-a1c1-f758861a6680)

MD and below (up to 768px), we switch to a mobile layout and show the mobile accordion
![image](https://github.com/user-attachments/assets/deb92a59-4f62-455a-885f-59e5463509cd)